addons/account_peppol/wizard/account_move_send.py

```diff
@@ -47,6 +47,12 @@ def _compute_checkbox_send_peppol(self):
         for wizard in self:
             wizard.checkbox_send_peppol = wizard.enable_peppol and not wizard.peppol_warning
 
+    @api.depends('enable_peppol')
```

Comment:
checkbox_send_peppol
Doing that will affect the flow and case unexpected behavior.
When we press a send&print, Peppol option will be the default, If we check on the email and then un-check peppol option then re-check it again, It will automatically uncheck the email option which is not what we need.

So, Depending on the enable_peppol is the good option to not allowing that as it also make sense because this was the place I put the logic in (at the very beginning)

Reply:
That's more a functional decision to know whether or not we want to re-trigger the compute when the checkbox changes or only at the launch of the wizard.
The dependency between "BIS Billing 3.0" and Peppol is for example based on the checkbox.
Then maybe the right `depends` is just `move_ids`. Really depends on what we want :woman_shrugging: 
But you're computation atm doesn't seem correct to me.

Reply:
What we want is just having send by peppol active & the send by email not by default at the launch of the wizard.
We don't want to have this behavior I described earlier in the comment above. So, Thinking based on that I don't see any dependencies that we should have on the extended computed method. As it should only been triggered at the load of the wizard and apply on the ones that have the email checkbox set!
Or Do you disagree with that?